install.rb

```diff
@@ -255,10 +255,13 @@ def install_binfile(from, op_file, target)
       tmp_file2 = Tempfile.new('hiera-wrapper')
       cwv = <<-EOS
 @echo off
-setlocal
-set RUBY_BIN=%~dp0
-set RUBY_BIN=%RUBY_BIN:\\=/%
-"%RUBY_BIN%ruby.exe" -x "%RUBY_BIN%hiera" %*
+SETLOCAL
+if exist "%~dp0environment.bat" (
+  call "%~dp0environment.bat" %0 %*
+) else (
+  SET PATH=%~dp0;%PATH%
+)
+ruby.exe -S -- hiera %*
```

Comment:
I'm not sure. I don't think so. Originally, this batch file assumes ruby is in the current directory. It's not obvious, but environment.bat installed in puppet-agent just sets path to include the current directory where ruby is. It's mainly set up like this to preserve functionality....